packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -333,18 +335,18 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     this.logger.sendErrorEvent({ eventName: "TreeLatest_SecondCall", hasClaims: !!tokenFetchOptions.claims });
                 }
 
+                // Choose min of host specified limits or driver specified limits.
+                const maxSnapshotSizeLimit = 25000000; // 25 MB
+                const maxSnapshotFetchTimeout = 30000; // 30 sec
                 const snapshotOptions: ISnapshotOptions = {
                     deltas: 1,
                     channels: 1,
                     blobs: 2,
                     ...this.hostPolicy.snapshotOptions,
```

Comment:
Please note that user settings overwrite defaults (above). I'll refer to it later.

Reply:
We are choosing the minimum of user and driver.

Reply:
My comment was to bring attention to other settings, like blobs=2. If user specifies other setting, then it takes over (that said, we should double check that there is a setting that client can supply, like 0, as undefined will not cut it - we will overwrite it).
I.e. user specifying blobs = 0 is preserved here, and thus falling back to trees later (if we timed out) may not make sense because we already were not asking for trees,